platform/windows/os_windows.cpp

```diff
@@ -212,6 +212,13 @@ void OS_Windows::initialize() {
 
 	process_map = memnew((Map<ProcessID, ProcessInfo>));
 
+	// Add current Godot PID to the list of known PIDs
+	ProcessInfo current_pi;
+	PROCESS_INFORMATION current_pi_pi;
```

Comment:
C:\projects\godot\platform\windows\os_windows.cpp(218) : error C2220: the following warning is treated as an error
C:\projects\godot\platform\windows\os_windows.cpp(218) : warning C4700: uninitialized local variable 'current_pi_pi' used

Reply:
Fixed